src/fiatExchanges/quotes/constants.ts

```diff
@@ -19,7 +19,47 @@ export const DEFAULT_ALLOWED_VALUES: DefaultAllowedValues = {
 export enum SettlementTime {
   // List of available settlement time strings for SelectProvider and
   // TransferStatus screens
-  LESS_THAN_ONE_HOUR = 'LESS_THAN_ONE_HOUR', // only for SelectProvider
-  LESS_THAN_24_HOURS = 'LESS_THAN_24_HOURS',
-  ONE_TO_THREE_DAYS = 'ONE_TO_THREE_DAYS',
+  LESS_THAN_ONE_HOUR = 'LESS_THAN_ONE_HOUR',
+  LESS_THAN_X_HOURS = 'LESS_THAN_X_HOURS',
+  X_TO_Y_HOURS = 'X_TO_Y_HOURS',
+  LESS_THAN_X_DAYS = 'LESS_THAN_X_DAYS',
+  X_TO_Y_DAYS = 'X_TO_Y_DAYS',
+}
+
+export type SettlementEstimation =
+  | {
+      settlementTime: SettlementTime.LESS_THAN_ONE_HOUR
+    }
+  | {
+      settlementTime: SettlementTime.LESS_THAN_X_HOURS
+      upperBound: number
+    }
+  | {
+      settlementTime: SettlementTime.X_TO_Y_HOURS
+      lowerBound: number
+      upperBound: number
+    }
+  | {
+      settlementTime: SettlementTime.LESS_THAN_X_DAYS
+      upperBound: number
+    }
+  | {
+      settlementTime: SettlementTime.X_TO_Y_DAYS
+      lowerBound: number
+      upperBound: number
+    }
```

Comment:
```suggestion
      settlementTime: SettlementTime.LESS_THAN_X_HOURS | SettlementTime.LESS_THAN_X_DAYS
      upperBound: number
    }
  | {
      settlementTime: SettlementTime.X_TO_Y_HOURS | SettlementTime.X_TO_Y_DAYS
      lowerBound: number
      upperBound: number
    }
```
code golf

Reply:
btw LESS_THAN_ONE_HOUR doesn't seem necessary as its own case. it's just a special case of LESS_THAN_X_HOURS

Reply:
actually I'm realizing we probly need it so we can use a singular "hour" instead of "hours". bummer

Reply:
might be worth leaving a code comment explaining that

Reply:
Yeah the singular hour is the reason for the extra enum key, added a comment